rmnetctl/src/Android.mk

```diff
@@ -30,8 +31,7 @@ LOCAL_MODULE := librmnetctl
 LOCAL_MODULE_TAGS := optional
 LOCAL_PRELINK_MODULE := false
 
-LOCAL_MODULE_PATH_32 := $(TARGET_OUT_VENDOR)/lib
-LOCAL_MODULE_PATH_64 := $(TARGET_OUT_VENDOR)/lib64
+LOCAL_VENDOR_MODULE: true
```

Comment:
```
vendor/qcom/opensource/dataservices/rmnetctl/src/Android.mk:34: error: writing to readonly directory: "LOCAL_VENDOR_MODULE"
```
You accidentally turned this one into a build rule :wink:

Reply:
^^

Reply:
Also, whaddaya think? https://github.com/ix5/vendor-qcom-opensource-dataservices/tree/q-mr1-nocopyheaders-androidbp

Reply:
You know I like `.bp` :smile: 

Let me apply some review inline as to not diverge it here.